tests/scripts/github-action-helper.sh

```diff
@@ -40,20 +40,20 @@ function print_k8s_cluster_status() {
 }
```

Comment:
Did only whitespace change in this file? If so, shall we revert it?

Reply:
reverted